src/components/Guide.js

```diff
@@ -1,365 +1,1377 @@
                   </section>
 
                   <section>
-                    <h1 id="topic-2">TOPIC 2</h1>
-
-                    <p>
-                      Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-                      sed do eiusmod tempor incididunt ut labore et dolore magna
-                      aliqua. Ut enim ad minim veniam, quis nostrud exercitation
-                      ullamco laboris nisi ut aliquip ex ea commodo consequat.
-                      Duis aute irure dolor in reprehenderit in voluptate velit
-                      esse cillum dolore eu fugiat nulla pariatur. Excepteur
-                      sint occaecat cu pidatat non proident, sunt in culpa qui
-                      officia deserunt mol lit anim id est laborum.
-                    </p>
+                    {/* Empty padding */}
+                    <Typography id="understanding_candlestick_charts">
+                      ‎
+                    </Typography>
+
+                    <Typography className={classes.topic} variant="h1">
+                      Part 2 Understanding Candlestick Charts
+                    </Typography>
+
+                    <section>
+                      {/* Empty padding */}
+                      <Typography id="history">‎</Typography>
+
+                      <Typography className={classes.subtopic} variant="h2">
+                        History
+                      </Typography>
+
+                      <Typography className={classes.paragraph} variant="body1">
+                        Candlestick charts are thought to have been developed in
+                        the 18th century by Munehisa Homma, a Japanese rice
+                        trader. They were popularized in the west by Steve
+                        Nison. According to him, it is likely that the original
+                        candlestick chart were modified and refined over many
+                        years of trading, eventually resulting in the system of
+                        candlestick charting that we use today.
+                      </Typography>
+                    </section>
+
+                    <section>
+                      {/* Empty padding */}
+                      <Typography id="explanation">‎</Typography>
+
+                      <Typography className={classes.subtopic} variant="h2">
+                        Explanation
+                      </Typography>
+
+                      <Typography className={classes.paragraph} variant="body1">
+                        Each candlestick chart contains four data sets, namely
+                        open, high, low and close values over a certain time
+                        period.
+                      </Typography>
+
+                      <img
+                        src="https://github.com/VisuTrader-Orbital2021/visutrader-frontend/blob/assets/images/guide/candlestick.svg"
+                        alt="candlestick"
+                      />
```

Comment:
Don't know why but the image doesn't render.

Reply:
More likely because we haven't push it to the main branch yet. I think we can include it directly from the `public` folder?

Reply:
Ok.